database/feegrant.go

```diff
@@ -0,0 +1,46 @@
+package database
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	feegranttypes "github.com/cosmos/cosmos-sdk/x/feegrant"
+	"github.com/forbole/bdjuno/v2/types"
+)
+
+// SaveFeeGrantAllowance allows to store the fee grant allowances for the given block height
+func (db *Db) SaveFeeGrantAllowance(allowance *feegranttypes.MsgGrantAllowance, height int64) error {
+	stmt := `
+INSERT INTO fee_grant_allowance(grantee, granter, allowance, height) 
+VALUES ($1, $2, $3, $4) 
+ON CONFLICT DO NOTHING`
```

Comment:
Once the constraint is implemented, we need to change this to make sure that if we're updating multiple heights at the same time, we do not override a grant on a higher height. To do this, we can do as we do in all other tables: 
```suggestion
INSERT INTO fee_grant_allowance(grantee, granter, allowance, height) 
VALUES ($1, $2, $3, $4) 
ON CONFLICT ON CONSTRAINT unique_fee_grant_allowance DO UPDATE 
    SET allowance = excluded.allowance,
        height = excluded.height
WHERE fee_grant_allowance.height <= excluded.height`
```